gpii/node_modules/packagekit/nodepackagekit/nodepackagekit_test.js

```diff
@@ -155,6 +155,17 @@ var fluid = require("universal"),
         return matchPkg;
     };
 
+    pkTests.testPerformActionFailure = function() {
```

Comment:
Not sure if that was directed at me.  My concern there would be cleanly avoid linting node_modules and other content that isn't part of the project.